src/com/google/javascript/jscomp/GlobalNamespace.java

```diff
@@ -1075,9 +1088,22 @@ boolean isSimpleStubDeclaration() {
     }
 
     boolean canCollapse() {
-      return !inExterns && !isGetOrSetDefinition() && (declaredType ||
+      boolean isAnnotatedNoCollapse = docInfo == null ?
+          false : docInfo.isNoCollapse();
+      if (!isAnnotatedNoCollapse) {
+        Ref ref = this.getDeclaration();
+        if (ref != null && ref.node != null) {
+          JSDocInfo info = NodeUtil.getBestJSDocInfo(ref.node);
```

Comment:
I would think that docInfo is always the same as info (the jsdoc at the assignment). Why is this not the case?